README.md

```diff
@@ -52,7 +51,7 @@ To install the package, follow these steps:
     ],
     ```
 
-3. Publish the package configuration:
+6. Publish the package configuration:
 
     ```bash
     php artisan vendor:publish --provider="Middleware\LaravelAPM\LaravelAPMServiceProvider"
```

Comment:
Should we change the order of `php artisan vendor:publish ....` and `config/laravel-apm.php` configuration ?

In a recent client call `php artisan vendor:publish ....`  was failing due to missing endpoint.